server/controllers/auth.js

```diff
@@ -52,10 +54,34 @@ exports.registerUser = asyncHandler(async (req, res, next) => {
         }
       }
     });
-  } else {
+  } else if(user && accountType == "petSitter") {
+    await sitterSchema.create({
+      userId: user._id,
+      name
+    });
+    console.log("sitter created.")
+    const token = generateToken(user._id);
+    const secondsInWeek = 604800;
+
+    res.cookie("token", token, {
+      httpOnly: true,
+      maxAge: secondsInWeek * 1000
+    });
+
+    res.status(201).json({
+      success: {
+        user: {
+          id: user._id,
+          name: user.name,
+          email: user.email
+        }
+      }
+    });
+  }else{
```

Comment:
Probably safe to say if it's not petSitter then it's going to be a petSitter. Logically I think it would make more sense to do `if (accountType == 'petSitter') { . . . }` then else would just create a new customer (Profile)